chart/compass/charts/gateway/values.yaml

```diff
@@ -5,6 +5,12 @@ deployment:
     runAsUser: 2000
     allowPrivilegeEscalation: false
 
-gateway:
```

Comment:
be very careful with such modifications, if someone overrides these values, your modification will break his code.

Reply:
I abandoned the modification.